prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -12,6 +12,49 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+locals {
+  sinker_monitoring_resources = {
+    pods: "sinker_pods_removed"
+    prowjobs: "sinker_prow_jobs_cleaned"
+  }
+}
+
+resource "google_monitoring_alert_policy" "sinker-alerts" {
+  project      = var.project
+  for_each      = local.sinker_monitoring_resources
+  display_name = "sinker-not-deleting-${each.key}"
+  combiner     = "OR" # required
+
+  conditions {
+    display_name = "Sinker not deleting ${each.key}"
+
+    condition_monitoring_query_language {
+      duration = "300s"
+      query    = <<-EOT
+      fetch k8s_container
+      | metric 'workload.googleapis.com/${each.value}'
+      | group_by 1h, [value_sinker_removed_sum: sum(value.${each.value})]
```

Comment:
nit: The second aggregation (an implicit sum) can be done by the first summation if you include `[], ` between `group_by` and `1h,` on this line.

Reply:
fixed in https://github.com/GoogleCloudPlatform/oss-test-infra/pull/1030